src/App.js

```diff
@@ -11,34 +13,181 @@ class App extends React.Component {
       cardDeck: makeShuffledDeck(),
       // currCards holds the cards from the current round
       currCards: [],
+      score: [0, 0],
+      roundsLeft: 26,
+      winner: "undecided",
+      gameEnded: false,
+      gamesWon: [0, 0],
     };
   }
 
+  resetGame = () => {
+    this.setState((state) => {
+      return {
+        cardDeck: makeShuffledDeck(),
+        // currCards holds the cards from the current round
+        currCards: [],
+        score: [0, 0],
+        roundsLeft: 26,
+        winner: "undecided",
+        gameEnded: false,
+      };
+    });
+  };
+
+  computeOutcome = () => {
+    this.setState((state) => {
+      let cards = state.currCards;
```

Comment:
try to only use `let` if you're going to change the variable's value in the same scope, otherwise default to `const`. using `let` implies that you intend to change the value, which may be distracting to a developer looking out for the change and not being able to find it